src/styles/modules/App.scss

```diff
@@ -15,17 +15,17 @@
     text-align: center;
 }
 
-.title {
+.app__title {
```

Comment:
let's go with one more nice scss feature, so the shorthands for scss. Lets write it as:
```scss
.app {
  &__title {
    rules...
  }
}
```
in the same rest of the css classes :)